packages/collabswarm/package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "tsc": "tsc -p tsconfig.json",
     "prepublishOnly": "npm run tsc",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest json-serializer.test.ts",
```

Does this need to include the specific test file? I'd prefer a command that runs all test files in the project

```suggestion
    "test": "jest",
```

---

Yes that should def happen as more tests are added. We can change now so as not to forget later. Was useful to save time during set up (5-10 s a run!), then had in mind a statement you made "keep the scope on what you need and add more later" about something about types probably